public/app/features/alerting/unified/Silences.tsx

```diff
@@ -30,9 +31,20 @@ const Silences = () => {
         <Route exact path="/alerting/silence/new">
           {({ location }) => {
             const queryParams = new URLSearchParams(location.search);
+            const potentialRuleUid = queryParams
+              .getAll('matcher')
+              .find((m) => m.startsWith(MATCHER_ALERT_RULE_UID))
```

Comment:
Should we use `parseMatcher` or any other similar helper here? 
The `alert_rule_uid` shouldn't contain any problematic characters, so the code should work, rather for consistency/readability